### DISCUSSION TITLE
Clarity website

### DISCUSSION THREAD

**Discussion by bbogdanov** [vmware-clarity/ng-clarity#223] [General]
Guys, what happened with the website?

Currently - https://clarity.design redirects to the Angular documentation

I see there is a https://core.clarity.design that shows the Clarity Core documentation

It should be good to have some kind of announcement in the intro page (maybe an alert) for that change. It is quite unpleasant to find myself searching around the docs and the repos to understand what happened.

**Comment by yusufozturk** [comment]
Thanks for starting this conversation @bbogdanov 

Additionally I wonder if ng-clarity is the first citizen now? I thought ng-clarity was going to be retired and clarity-core was going to be the main project, so people can use it with React etc. Is this change means, you will continue to invest in angular again?

I wonder if VMware products like vSphere, vRealize etc uses ng-clarity or clarity-core.. We are checking them time to time, but they are not consistent with Clarity design rules and sometimes they even use different color options or buggy UI elements. It's very confusing to understand what is the primary project here. And I don't understand why sometimes different VMware products use different UI elements which is not consistent with clarity design rules.

For example, vCenter is getting closer and closer to clarity UI rules with each updates but there are still some inconsistencies. So I assume they don't use ng-clarity or clarity-core but instead they contribute into their own version?

**Reply by ashleyryan** [reply]
The reality is that the vast majority of VMware products are using ng-clarity, and adoption to Core has been slower than anticipated. Ng-clarity will certainly be getting more love going forward than we had previously anticipated, but new development in NG is unclear. There are non-angular teams that can't use NG, so we continue to support them with Core, and we're doing work (like with the shim) to bring their colors and styles more in line so they can be used alongside each other to complement each other.

Most of the products you mentioned are using ng-clarity, and many of those products existed and had their own styling before Clarity was a thing - they all have their own designers, and we are not, nor can we be, the UI Police, telling people what to do. They do all use Clarity, but they may either be on older versions of Clarity, or have remaining legacy code/styling/design.
